index.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express')
+const app = express();
+const cors = require('cors');
+const quotes = require('./quotes.json');
+
+app.use(express.json());
+app.use(cors());
+
+const port = process.env.PORT || 3000;
+
+
+function getRandomQuote(num) {
+  return quotes[num];
+}
+
```

Comment:
Is this definitely the right file for this project? Looks good and very efficient for retrieving items from the array. I also like your try-catch on line 32